app/javascript/Components/groups_manager.jsx

```diff
@@ -563,6 +563,31 @@ class RawGroupsTable extends React.Component {
       filterable: false,
       sortMethod: durationSort,
     },
+    {
+      Header: "Penalty",
+      id: "late_penalty_applied",
+      accessor: "extension",
+      Cell: row => {
+        return row.original.extension.apply_penalty ? (
+          <FontAwesomeIcon icon="fa-solid fa-square-check" />
+        ) : (
+          <FontAwesomeIcon icon="fa-solid fa-xmark" />
+        );
+      },
+      minWidth: 30,
+      filterable: false,
+      sortMethod: (a, b) => {
```

Comment:
Given the small size, simplicity and one time use of this function, I thought it'd be a good idea to implement directly in the group_manager file. Let me know if it'd be better to place this sorting function inside the [table_helpers](https://github.com/MarkUsProject/Markus/blob/master/app/javascript/Components/Helpers/table_helpers.jsx) file instead.